content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -427,7 +427,7 @@ We'll guide you through how to create and update applications with Deployments.
 Let's say you were running version 1.14.2 of nginx:
 
 ```shell
-kubectl run my-nginx --image=nginx:1.14.2 --replicas=3
+ kubectl create deploy  my-nginx --image=nginx:1.14.2
```

Comment:
```syggestion
kubectl create deployment my-nginx --image=nginx:1.14.2
```

This is so that readers understand that it creates a Deployment because of the resource name given, and that `deploy` is not a special keyword.